packages/app-mobile/components/screens/Note.tsx

```diff
@@ -736,8 +728,22 @@ class NoteScreenComponent extends BaseScreenComponent {
 	}
 
 	async attachPhoto_onPress() {
-		const response = await this.showImagePicker({ mediaType: 'photo', noData: true });
-		await this.attachFile(response, 'image');
+		// the selection Limit should be specfied. I think 200 is enough?
+		const response: ImagePickerResponse = await this.showImagePicker({ mediaType: 'photo', includeBase64: false, selectionLimit: 200 });
+
+		if (response.errorCode) {
+			reg.logger().warn('Got error from picker', response.errorCode);
+			return;
+		}
+
+		if (response.didCancel) {
+			reg.logger().info('User cancelled picker');
+			return;
+		}
+
+		for (const asset of response.assets) {
+			await this.attachFile(asset, 'image');
```

Comment:
I wanted to implement bulk attachment at first, but I checked the code and I think it needs a lot of changes. I'm afraid of making too much changes because I have no experience of Typescript.
Also, I think bulk attachment might be risky? If the app fails to load one of the files, the whole process will go wrong, or we need to implement some extra mechanism to handle this situation. Anyway, I think it's better for you collaborators to make the decision.

Reply:
I don't have any idea about how to implement the bulk attachment due to my innocence of Typescript.

Reply:
@roman-r-m Do we  need to implement bulk attachment in this PR?

Reply:
(Talked in this [topic](https://discourse.joplinapp.org/t/mobile-attach-multiple-files-at-once/27231))